cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/UnmanagedClusterInventory.tsx

```diff
@@ -1,32 +1,72 @@
-                <div cds-layout="vertical col:8 gap:lg">
+                <div cds-layout="vertical gap:md gap@md:lg col@sm:8 col:8">
                     <div cds-text="title">
-                        <CdsIcon cds-layout="m-r:sm" shape="block" size="xl" className="icon-blue"></CdsIcon>
-                        Unmanaged Cluster Inventory
+                        <CdsIcon cds-layout="m-r:sm" shape="computer" size="xl" className="icon-blue"></CdsIcon>
+                        Unmanaged Cluster
                     </div>
-                    <div cds-text="subsection" cds-layout="p-y:md">
-                        Placeholder page for unmanaged cluster inventory
+                    <div cds-text="subsection">
+                        Create a single node, local workstation cluster suitable for a development/test environment. It requires minimal
+                        local resources and is fast to deploy. It provides support for running multiple clusters. The default Tanzu
+                        Community Edition package repository is automatically installed when you deploy an unmanaged cluster.
+                    </div>
+                    <div cds-layout="vertical gap:md gap@md:lg col@sm:12 col:12">
+                        <CdsButton className="cluster-action-btn" status="primary" onClick={() => navigate(NavRoutes.DEPLOY_PROGRESS)}>
+                            <CdsIcon shape="cluster"></CdsIcon>
+                            Create Unmanaged Cluster
+                        </CdsButton>
+                    </div>
+                    <div cds-layout="vertical gap:lg col:6">
+                        <div cds-layout="vertical gap:md gap@md:lg col@sm:12 col:12">
+                            <CdsAlertGroup status="success">
+                                <CdsAlert>This is an alert with a status</CdsAlert>
+                            </CdsAlertGroup>
+                        </div>
+                        {unmanagedClusters.map((data, index) => {
+                            return <UnmanagedClusterInfo key={index} name={data.name} provider={data.provider} status={data.status} />;
+                        })}
                     </div>
                 </div>
                 <div cds-layout="col@sm:4 col:4 container:fill">
-                    <CdsButton
-                        action="flat"
-                        onClick={() => {
-                            window.open('https://tanzucommunityedition.io/docs/v0.11/architecture/#unmanaged-clusters', '_blank');
-                        }}
-                    >
-                        Learn more about Unmanaged Clusters
-                    </CdsButton>
+                    <div cds-layout="vertical">
+                        <CdsButton
+                            action="flat"
+                            onClick={() => {
+                                window.open('http://tanzucommunityedition.io', '_blank');
+                            }}
+                        >
```

Comment:
I think we may want a deeper link to a page focused on unmanaged clusters

Reply:
Thank you for the feedback I think I was told that right section of the screen would be filled with more specific information in the future. But I do not have access to any of that information. If you have an ideas or know what goes there I would be happy to add it!